sections/Experiment.tex

```diff
@@ -37,29 +38,50 @@ \subsection{Evaluation Metrics}\label{sec:experiment_metrics}
 	\item Aggregation of confirmation measures
 \end{enumerate}
 \vejleder[inline]{hvad er intuitionen? hvorfår gør man det på denne måde?}
-\begin{equation}
-	\overrightarrow{v}(W') = \left\{ \sum_{w_i \in W'} \text{NPMI}(w_i, w_j)^\gamma \right\}_{j=1,\dots,|W|}
+The equations, explanations and values of the hyperparameters in this section are based on~\citet{Syed2017coherence} and the \textit{gensim} python package\footnote{\url{https://radimrehurek.com/gensim/}}.
+
+For segmentation a set of word pairs $S$ is created, which pairs each word in the top-N most probable words $W$ in a specific topic $t$ with all the other words in $W$.
+$S$ is defined by \autoref{eq:set_of_word_pairs}.
+
+\begin{equation}\label{eq:set_of_word_pairs}
+	S = \{(W', W*)|W' = \{w_i\};w_i \in W;W* = W\}
 \end{equation}
-\begin{equation}
-	\text{NPMI}(w_i,w_j)^\gamma = \left( \frac
-	{log\frac{P(w_i,w_j) + \epsilon}{P(w_i)\cdot P(w_j)}}
-	{-log(P(w_i,w_j) + \epsilon)} \right)
+
+Before calculating word probabilities, a sliding window of size $s$ where $s =110$ is used to create a set of sub documents $D_s$ over the document set $D$.
+These sub documents are used rather than the normal documents to capture some degree of words proximity.
+Word probabilities are calculated based on how many documents within the document set $D_s$ that they occur in.
+$p(w_i)$, is the number of sub documents in which the word $w_i$ occurs divided by $|D_s|$.
+$p(w_i, w_j)$, is the number of sub documents in which both words occur divided by $|D_s|$. 
+
+We create a \gls{npmi} matrix of size $|W|\times|W|$, with one entry word pair combination in $W$.
+\begin{equation}\label{eq:coherence_2}
+	\text{NPMI}(w_i,w_j) =  \frac{\log\frac{p(w_i,w_j) + \epsilon}{p(w_i)\cdot p(w_j)}}{-log(p(w_i,w_j) + \epsilon)}
 \end{equation}
-\begin{equation}
+\noindent where $\epsilon$ is a low number ($10^{-12}$) used to avoid $log(0)$.
+The \gls{npmi} matrix describes how much each word occurs alone versus together with other words.
+%This is a -1 to 1 normalization.
+
+After having calculated the \gls{npmi} matrix, we construct context vectors for both elements $W'$ and $W*$ in each word-pair $S_i$, by summing over the rows of the \gls{npmi} matrix.
+This summation describes how much each top word co-occurs with the other words in $W$.
```

Comment:
Jeg har svært ved, igennem teksten her, at forestille mig hvordan disse vektorer ser ud. Der mangler måske noget for at gøre beskrivelsen tydelig, men jeg ved ikke hvad, da jeg ikke har læst topic coherence artiklen endnu.

Reply:
jeg ved ikke helt hvad jeg skal skrive tbh :/

Reply:
Det er i orden. Vi kan lade den ligge, og så måske kigge på det senere i skriveprocessen.